### DISCUSSION TITLE
Delete :/

### DISCUSSION THREAD

**Discussion by Khaled-Khedr** [Melledy/Grasscutter#1206] [Support]
C:\Users\Dark\Downloads\DARK Server\Grasscutter-development>account create dark 1
'account' is not recognized as an internal or external command,
operable program or batch file.

**Comment by Lumi-nary** [comment]
Bruh have you started the server first?

**Reply by Khaled-Khedr** [reply]
yes

**Reply by Lumi-nary** [reply]
use the cmd/shell you just opened to create server and put the command account create there

**Comment by nitrog0d** [comment]
sir you have to be running grasscutter and put that command in the shell you opened it

**Comment by gawgua** [comment]
![image](https://user-images.githubusercontent.com/104498646/172128131-a34f965f-4056-4d9f-b3bc-eeb522e2988e.png)
sir, put in the console that you used to run grasscutter

**Reply by nitrog0d** [reply]
my fellow sir please spend some time and read the wiki

**Reply by Khaled-Khedr** [reply]
java -jar grasscutter.jar didnt work either

**Reply by gawgua** [reply]
> java -jar grasscutter.jar didnt work either

it is not grasscutter.jar if you havent rename it, read the file name again
💀

**Reply by Khaled-Khedr** [reply]
ffs thank you gawgua it was the fucking name issue and I didnt notice it

**Reply by nitrog0d** [reply]
LETS GOOOOOO

**Comment by danr789** [comment]
If You had taken time to read both the README.md and the wiki, it would've helped, but nvm, inside the directory where the jar file is, run ``java -jar grasscutter.jar -s`` to start the server, then to create an account, in the SAME command prompt, run ``account add <username> <uid>``
